### DISCUSSION TITLE
AOI instead of TMS boundaries

### DISCUSSION THREAD

**Discussion by canunen** [hotosm/fAIr#241] [Ideas]
Hi fAIr folks, 

For the case of multiple TMSses on OAM in case of larger areas, we were discussing the utilization of Kontur's mosaic TMS, but then the mosiac is covering everything there is in OAM. 

So, would specifiying an AOI (Json of freehand) to restrict where the predictions should happen in be something to consider adding? 

Cheers.

**Comment by kshitijrajsharma** [comment]
Hi ! 

I am not sure Kontur's mosaic is good enought to run prediction on but def we are looking for option that enabels user to run prediciton on different images ! Currently defining area of interest is not possible using UI but there are scripts that can do that ! Thanks for bringing this to attention , I will update the thread with related issues and updates as they come

**Comment by omranlm** [comment]
Thanks @canunen  for the topic, Actually I have just starting testing that scenario and let's test it together 
Technically you as end user won't be able to create the dataset using Kontur mosaic as it needs to have .png at the end of the TMS link, however, technically I can do a minor change from the back end to create a dataset and already did 
https://fair-dev.hotosm.org/training-datasets/218
![image](https://github.com/hotosm/fAIr/assets/39491329/254e9f77-c84b-4d76-a82d-5fb0e1487834)

I think created a dummy model and started its training. If it works find, it might be a potential solution with minor changes from our end to be able to use multiple imageries but through Kontu's mosaic 

The model is https://fair-dev.hotosm.org/ai-models/153

P.S. kinfly note  that the dataset I provided here is not accurate at all and it is for the purpose of testing using mosaic link

**Reply by omranlm** [reply]
My initial observation is that Kontur's mosaic link is a bit slower than OAM imagery loading but not a blocker
